node-test/src/test/scala/org/bitcoins/node/NeutrinoNodeWithWalletTest.scala

```diff
@@ -73,8 +71,7 @@ class NeutrinoNodeWithWalletTest extends NodeTestWithCachedBitcoindNewest {
     //unconfirmed = 3 BTC - TestAmount - TestFees
     val condition1 = () => {
       condition(
-        expectedConfirmedAmount = 3.bitcoin,
-        expectedUnconfirmedAmount = 3.bitcoin - TestAmount - TestFees,
+        expectedBalance = 6.bitcoin - TestAmount - TestFees,
```

Comment:
we are able to change this because we never change the `expectedConfirmedAmount`. This needed to change because sometimes we would select different size utxos and it would change how much sats become confirmed vs unconfirmed